attendance.php

```diff
@@ -444,6 +444,11 @@ function get_user_course_id($email) {
 	}
 
 	$is_staff = strpos($_SESSION["user"], "@bebraven.org") !== FALSE || strpos($_SESSION["user"], "@beyondz.org") !== FALSE;
+        // For test accounts, treat them as users, not staff, b/c we want to mimic the user experience with test accounts
+        // and if we need to do staff stuff, we should use actual bebraven.org accounts.
+        // Example test account: brian+testblah@bebraven.org
+	if (preg_match('/\+test.*@bebraven.org/', $_SESSION["user"]))
+          $is_staff = false;
```

Comment:
I know this works, but I am always afraid of using single line if statements that aren't wrapped. Not that it's bad code, but the person coming behind usually misses this and causes errors. 

Not change requested just me speaking aloud!